samples/CSharp/FormatSolution/FormatSolution.csproj

```diff
@@ -14,9 +14,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" Version="15.3.409" />
-    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.3.409" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.6.0" />
+    <PackageReference Include="Microsoft.Build.Locator" Version="1.0.13" />
+
+    <!-- These package versions should be updated to 2.9.0 when released on NuGet -->
+    <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.MSBuild" Version="2.9.0-beta4-62810-05" />
```

Comment:
Hmm - it looks like we screwed up the versioning of 2.9 already.  We probably don't want these packages to have `-beta4` in them.

@jaredpar @jasonmalinowski ?

Reply:
I have no idea. Those are the packages on the roslyn feed.

Reply:
Ah, cute. Uh, I guess we could bump this back to -beta1 if we wanted to. @brettfo?

Reply:
what if we need beta4 in 15.8?